windows/ReactTestApp/MainPage.h

```diff
@@ -8,10 +8,22 @@ namespace winrt::ReactTestApp::implementation
     public:
         MainPage();
 
-        Windows::Foundation::Collections::IVector<ReactTestApp::ComponentViewModel> Components();
+        void LoadFromJSBundle(Windows::Foundation::IInspectable const &,
+                              Windows::UI::Xaml::RoutedEventArgs);
+        void LoadFromDevServer(Windows::Foundation::IInspectable const &,
+                               Windows::UI::Xaml::RoutedEventArgs);
+        void SetComponents();
 
     private:
-        Windows::Foundation::Collections::IVector<ReactTestApp::ComponentViewModel> m_components;
+        winrt::Microsoft::ReactNative::ReactNativeHost m_reactNativeHost;
+        std::vector<ReactTestApp::ComponentViewModel> m_components;
```

Comment:
Do we still need this? If we merge `SetComponents` and `InitReact`, I think it's unnecessary.